src/components/AboutPage/components/AboutList/components/AboutItem/components/AboutItemActions/AboutItemActions.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CardActions, Button } from '@mui/material';
+import type { AboutContacts } from '../../../../../../types/About';
+import './AboutItemActions.scss';
+
+interface AboutItemActionsProps {
+  contacts: AboutContacts[];
+}
+
+export const AboutItemActions = ({ contacts }: AboutItemActionsProps) => (
```

Comment:
I think we decided to write in this way: 
`export const AboutItemActions React.FC<AboutItemActionsProps>  = ({ contacts }: AboutItemActionsProps) .....`

Reply:
I didn't use this in the Astronauts page.
Maybe i missed something...